jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -8253,7 +8253,7 @@ $as_echo "$DEBUG_LEVEL" >&6; }
   elif test "x$OPENJ9_CPU" = xppc-64; then
     OPENJ9_PLATFORM_CODE=ap64
   else
-    as_fn_error $? "Unsupported OpenJ9 cpu ${OPENJ9_CPU}, contact support team!" "$LINENO" 5
```

Comment:
This change was made in custom-hook.m4 in previous commit. However, generated-configure.sh is not generated with that change. When I run autogen.sh now to generate generated-configure.sh files, it got updated.